python/pyproject.toml

```diff
@@ -0,0 +1,8 @@
+
+[build-system]
+requires = ["setuptools>=40.8.0", "wheel", "cmake>=3.18"]
+
+[tool.autopep8]
+aggressive = 1
+ignore = "E501,E701,E731,W690"
```

Comment:
This W690 check does a python 2->3 conversion which adds `from functools import reduce` if you use the global name "reduce" anywhere.